share/spice/meetup/meetup.js

```diff
@@ -0,0 +1,45 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_meetup = function(api_result){
+
+        if (!api_result || api_result.error || !api_result.results) {
+            return Spice.failed('meetup');
+        }
+
+        Spice.add({
+          id: "meetup",
+          name: "meetup",
+          data: api_result.results,
+
+          meta: {
+              count: api_result.results.length,
+              itemType: 'meetups',
+              sourceName: "Meetup.com",
+              sourceUrl: 'http://www.meetup.com/'
```

Comment:
Can we make the "More at" link to what the user is looking for like http://www.meetup.com/find/events/?allMeetups=true&radius=10&userFreeform=phi&mcId=c19019&change=yes for Philly?